frontend/src/app/category-chips/category-chips.component.html

```diff
@@ -1,8 +1,8 @@
 <mat-chip-list>
   <mat-chip
     *ngFor="let c of categories"
-    [routerLink]="[ '/search/filter' ]"
-    [queryParams]="{'words': c.category.name}"
+    [routerLink]="[ '/search' ]"
+    [queryParams]="{'category': c.category.id}"
```

Comment:
It feels really good to see this fixed! I was really dissatisfied with how I'd left it.